gap/TGQuotientSequences.gd

```diff
@@ -25,6 +25,130 @@
+#!   components:
+#!   - <Ref Oper='Signature' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     signature of the underlying triangle group
+#!   - <Ref Oper='BoundByGenus' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     upper bound of the triangle group quotient genus
+#!   - <Ref Oper='GetListTGQuotients' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     list of quotients with genus smaller than upper bound of the triangle group quotient genus
+#!   - <Ref Oper='MirrorSymmetries' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     binary list, 1 if quotients have mirror symmetries 0 otherwise
+#!   - <Ref Oper='IsSparse' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     boolean, if true the adjacency matrix is sparsely represented
+#!   - <Ref Oper='AdjacencyMatrix' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     adjacency matrix, which includes all quotients with and without mirror symmetries
+#!   - <Ref Oper='NearestNeighborAdjacencyMatrix' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     adjacency matrix of normal subgroup relation between consecutive translation groups of corresponding quotients
+#!   - <Ref Oper='LongestSequence' Label='for TGQuotientSequencesAdjacencyMatrix' />:
+#!     list of longest sequence of quotient groups 
+#!   and is printed in the form
+#!   @BeginLog
+#!   TGQuotientSequencesAdjacencyMatrix( 
+#!     [r, q, p], 
+#!     boundByGenus = int,
+#!     lstTGQuotients = [[genus, number], ... ],
+#!     lstMirrorSymmetries = [ 0 / 1 , ... ],
+#!     sparse = bool,
+#!     adjMatrix = [[ ... ], ... ]
+#!   )
+#!   @EndLog
+  
+DeclareCategory( "IsTGQuotientSequencesAdjacencyMatrixObj", IsObject );
+
+#! @Description
+#!   Construct the adjacency matrix for the triangle group <A>tg</A> ($\Delta^+$), given as `ProperTriangleGroup` object (see
+#!   <Ref Sect="Section_TriangleGroups"/>), for available quotients from the library, i.e., Conder’s list.
+#!
+#!   The option `boundByGenus`, an upper bound, can be passed, which takes a positive integer below 102, limiting the number of 
+#!   quotients considered. If a positive integer below 102 is provided, triangle group quotients with genus smaller than 
+#!   `boundByGenus` will be used, a value above 102 will default back to 102. The default is 102. 
+#!   The option `sparse`, which takes a boolean, can be used to generate a sparse representation of the adjacency matrix.
+#!   If sparse is true the adjacency matrix is of the form `[ [ [ rowIdx, colIdx ], entry ], ... ]`, where `entry`
+#!   is the corresponding matrix entry at position `rowIdx` and `colIdx`, which represent indices of the matrix 
+#!   rows and columns, respectively. The default is false.
+#!
+#!   This function saves constrcuted translation groups in cache, which can be flushed
+#!   by calling <Ref Func="FlushCaches" BookName="ref"/>.
```

Comment:
This reference does not seem to resolve...
<img width="584" alt="image" src="https://github.com/user-attachments/assets/f501f8fe-5b32-498b-b2f6-017c89bac4d2">